<launch>

    <!--这里尽量不要出现中文路径-->

    <arg name="bin_dir" default="/media/cyber-z/E/test/kitti-test/original" /> 
    <arg name="pcd_dir" default="/media/cyber-z/E/test/t" /> 
    <arg name="label_dir" default="/media/cyber-z/E/test/label_tran" /> 
    <arg name="input_type_option" default="pcd" />       <!--  bin/pcd  -->
    <arg name="show_type_option" default="single" />     <!--  single  -->
    <arg name="show_count" default="1" />                <!--  single=1=default,start form show_num    -->
    
    <arg name="show_num" default="8" />
    <arg name="cloud_type" default="XYZ" />              <!--  XYZ/XYZI  不准备使用,预留接口-->


    <node pkg="kitti" type="box_show" name="kitti_show_c_trancood" output="screen">

        <param name="bin_dir" type="string" value="$(arg bin_dir)" />
        <param name="pcd_dir" type="string" value="$(arg pcd_dir)" />
        <param name="input_type_option" type="string" value="$(arg input_type_option)"/>
        <param name="show_type_option" type="string" value="$(arg show_type_option)" />
        <param name="show_count" type="int" value="$(arg show_count)" />
        <param name="show_num_" type="string" value="$(arg show_num)" />
        <param name="cloud_type" type="string" value="$(arg cloud_type)" />

    </node>


    <node pkg="kitti" type="kitti_show_p_trancood.py" name="kitti_show_p_trancood" output="screen">

        <param name="label_dir" type="string" value="$(arg label_dir)" />
        <param name="show_count" type="int" value="$(arg show_count)" />
        <param name="show_num" type="int" value="$(arg show_num)" />

    </node>

</launch>